miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -9,28 +9,41 @@ HotTarget.pool = {
   createColumns : function(config, create, data) {
     return [
         {
+          header : 'Pool Name',
+          data : 'name',
+          readOnly : true,
+          include : true,
+          unpackAfterSave : true,
+          unpack : function(pool, flat, setCellMeta) {
+            flat.name = pool.name;
+          },
+          pack : function(pool, flat, errorHandler) {
+            pool.name = flat.name;
```

Comment:
I noticed that, but is there a reason behind it? I assumed it was that it wouldn't have any actual effect. The service layer is responsible for that decision though.

We often have bugs due to field values getting lost somewhere in DB -> Hibernate -> DTO -> HOT -> DTO -> Hibernate -> DB transformations. Without good reason to exclude it, including the field everywhere seems preferable to me.

I can still remove it if you'd prefer. Just curious as to why?